app/classifier/tasks/text/index.jsx

```diff
@@ -110,7 +111,7 @@ export default class TextTask extends React.Component {
             onChange={this.handleChange}
             ref={this.textInput}
             rows={this.state.rows}
-            style={{ lineHeight: `${LINEHEIGHT}px` }}
+            style={{ lineHeight: `${LINEHEIGHT}px`, overflow: 'hidden' }}
```

Comment:
If the input text is very long, will this prevent it from being scrolled to edit it?

Reply:
You can still use the cursor and arrow keys to scroll down, but there isn't a scroll bar. The following screen shot has several lines of text below what is visible:
![screen shot 2018-07-25 at 2 24 22 am](https://user-images.githubusercontent.com/12118751/43186467-52d14630-8fb4-11e8-8020-7c97b0793bdd.png)

I'm not sure if setting a max number of rows (could be something more obnoxious, like 20) and risk text cutoff without scroll bar (like above) or not setting any max and letting expand as intended (with this PR hopefully fixing noted related bug) preferable.

Reply:
To be honest, a vertical scroll bar changes the width of an element, so I’m not really sure what was happening with `scrollHeight` here.